client/src/Components/PrivateRoutes/AuthenticatedRoutes.js

```diff
@@ -11,19 +13,41 @@ import Wizard4 from "../Wizard4/Wizard4";
 import ProtectedTicketRoute from "./ProtectedTicketRoute";
 
 class AuthenticatedRoutes extends Component {
-  render() {
+  constructor() {
+    super();
+    this.state = {
+      load: true
+    };
+
+  }
+
+  componentDidMount() {
+    axios.get('/api/user')
+      .then(res => {
+        this.props.setUserID(res.data.user_id)
+        this.props.setUserRole(res.data.user_role)
+        this.setState({load: false})
+      })
+      .catch(err => console.log(err))
+  }
+
+
+  render() {  
+    if (this.state.load === true) {
+      return <h2>Loading...</h2>
+    }
     return this.props.userID ? (
       <Switch>
-        <Route exact path="/" component={HomeView} />
+        <Route exact path="/tickets" component={HomeView} />
```

Comment:
Donovan mentioned before that for readability and that the homeView should be '/' and login '/login' (I had it switched too)

Reply:
i tried that too. we may need to look at the logic then because it would get stuck displaying a loading notification and never redirect to login. let me try something on this though